Dockerfile

```diff
@@ -23,9 +23,16 @@ WORKDIR /app
 
 COPY --from=builder ["/build/grype-server/grype-server", "./grype-server"]
 
+ENV DB_ROOT_DIR=/data
```

Comment:
It would be a good idea to change the default in the code too. Right now its set to /app which was based on the old Dockerfile. We can change the default in the code to `.` / the current directory so that if you run grype-server outside of docker it doesn't try to look for a directory that doesn't exist.